repl/evaluator.go

```diff
@@ -697,6 +730,12 @@ func (e *Evaluator) setOption(args []string) error {
 			if err != nil {
 				issues = append(issues, fmt.Sprintf("container: %v", err))
 			}
+		} else if arg == "--extension" {
```

Comment:
nit: You could consider making this a `switch`

Reply:
Done